var/spack/repos/builtin/packages/tensorflow/package.py

```diff
@@ -0,0 +1,291 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+from glob import glob
+
+
+class Tensorflow(Package):
+    """TensorFlow is an Open Source Software Library for Machine Intelligence
+    """
+
+    homepage = "https://www.tensorflow.org"
+    url      = "https://github.com/tensorflow/tensorflow/archive/v0.10.0.tar.gz"
+
+    version('2.0.0',        sha256='49b5f0495cd681cbcb5296a4476853d4aea19a43bdd9f179c928a977308a0617')
+    version('1.15.0',       sha256='a5d49c00a175a61da7431a9b289747d62339be9cf37600330ad63b611f7f5dc9')
+    version('1.14.0',       sha256='aa2a6a1daafa3af66807cfe0bc77bfe1144a9a53df9a96bab52e3e575b3047ed')
+    version('1.13.2',       sha256='abe3bf0c47845a628b7df4c57646f41a10ee70f914f1b018a5c761be75e1f1a9')
+    version('1.13.1',       sha256='7cd19978e6bc7edc2c847bce19f95515a742b34ea5e28e4389dade35348f58ed', preferred=True)
```

Comment:
```suggestion
    version('1.13.1',       sha256='7cd19978e6bc7edc2c847bce19f95515a742b34ea5e28e4389dade35348f58ed')
```

Reply:
I would be inclined to keep it preferred until we can compile v1.14 and v2.0 with cuda, which wasn't the case last time I checked.